server/index.js

```diff
@@ -36,3 +38,38 @@ app.get('/:id', (req, res) => {
     }
   });
 });
+
+
+app.post('/hotel/:hotel', (req, res) => {
+  let reviewInfo = req.body;
+  db.save(reviewInfo)
+  .then(result => {
+    res.send(result);
+  })
+  .catch((err) => {
+    console.log(err);
+  });
+})
+
+app.put('/hotel/:reviewId', (req, res) => {
+  let updateInfo = {};
+  updateInfo.id = req.params.reviewId;
+  updateInfo.newText = req.body.text;
+  db.update(updateInfo)
+  .then(result => {
+    res.send(result);
+  })
+  .catch((err) => {
+    console.log(err);
+  });
+});
+
+app.delete('/hotel/:reviewId', (req, res) => {
+  db.deleteRev(req.params.reviewId)
+  .then(result => {
+    res.send(result);
+  })
+  .catch((err) => {
+    console.log(err);
+  });
+})

```

Comment:
### These endpoints look great!
None of them are referring to database specific functions explicitly in the endpoint code blocks, which is perfect. You'll be able to easily swap out the `db` definition to plug in your other DBMS later in the project.